src/python/fuzzing/corpus_manager.py

```diff
@@ -25,7 +25,7 @@
 
 try:
   from google_cloud_utils import gsutil
-  DEFAULT_GSUTIL_RUNNER = gsutil.GSUtilRunner
+  DEFAULT_GSUTIL_RUNNER = gsutil.GSUtilRunner  # pylint: disable=invalid-name
```

Comment:
i'm a little surprised this gave an error. what was it complaining about?

Reply:
It was complaining all caps names aren't valid class names. I'm frankly surprised it figured this out too.

Reply:
wow nice. Let's just fix this to DefaultGSUtilRunner then

Reply:
Err....If I do that then pylint will complain that the assignment to `None` is badly named. I could refactor so that the assignment happens outside of try except, but I think it makes most sense to disable the check here.

Reply:
sgtm